codegen/templates/endpoint.tmpl

```diff
@@ -214,18 +202,18 @@ func (w {{$workflow}}) Handle(
 	}
 
 	// Filter and map response headers from client to server response.
-	endRespHead := map[string]string{}
-	{{if (ne (len $resHeaderMap) 0) -}}
+
+	// TODO: Add support for TChannel Headers with a switch here
+	resHeaders := zanzibar.ServerHTTPHeader{}
```

Comment:
Ideally a workflow should be transport agnostic without a switch.

Maybe we need `zanzibar.ServerHeader{}` type ?